main.tf

```diff
@@ -37,13 +38,22 @@ resource "aws_redshift_cluster" "this" {
 
   tags = "${var.tags}"
 
+  # Enhanced VPC routing
+  enhanced_vpc_routing = "${var.enhanced_vpc_routing}"
+
   lifecycle {
```

Comment:
Move `tags` and `lifecycle` arguments at the end as they were before.

Reply:
Done